DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: qmongr
 Title: Lightweight Quality Assessment of Norwegian Hospitals 
-Version: 0.5.0.9000
+Version: 0.6.0
```

Comment:
Should we do this directly on the master branch afterwards? How it is done in 04698d380c340

Reply:
At this stage it is probably OK to do it as you did it here.